### PR TITLE
Detect sudo invocations of Makefile #164

### PR DESCRIPTION
**Pull request by yfei1**
Resolve issue #164

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -32,6 +32,15 @@
 # make delete-gke-cluster
 #
 ## http://makefiletutorial.com/
+define n
```
Comment (jeremyje) [inline Makefile]: What's this for?

Comment (yfei1) [reply Makefile]: It is used to define a newline in the Makefile to improve error message readability. I look around but could not find a second way to do it.

**Comment by jeremyje** [comment]
You need to sync this PR.

File: Makefile (hunk 1 above)
Comment (jeremyje) [reply Makefile]: I think the diff is off here. Bad merge?

**Pull request merged**